### PR TITLE
Fix lerna bootstrap

### PR DESCRIPTION
**Pull request by SCG82**
Appears to be a known issue: https://github.com/lerna/lerna/issues/2352

Possibly a result of NPM's [flattening](https://medium.com/pnpm/pnpms-strictness-helps-to-avoid-silly-bugs-9a15fb306308)

#### ~~TODO~~ (done)
- ~~look into adding `--global-style` to  **npmClientArgs** in `lerna.json`~~

```bash
cd packages
for d in *; do
  echo $d
  ls -ghG $d/node_modules/@wdio/ 2>/dev/null
  ls -ghGd $d/node_modules/webdriver* 2>/dev/null
  ls -ghGd $d/node_modules/devtools 2>/dev/null
done
```

Before (some packages are not symlinked):
```ls
devtools
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:13 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
eslint-plugin-wdio
node_modules
wdio-allure-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-appium-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-appium-service/node_modules/webdriverio -> ../../webdriverio
wdio-browser-runner
total 0
lrwxrwxrwx 1 21 Jan  4 20:13 globals -> ../../../wdio-globals
lrwxrwxrwx 1 26 Jan  4 20:13 local-runner -> ../../../wdio-local-runner
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 29 Jan  4 20:13 mocha-framework -> ../../../wdio-mocha-framework
lrwxrwxrwx 1 23 Jan  4 20:13 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 20 Jan  4 20:13 runner -> ../../../wdio-runner
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 15 Jan  4 20:13 wdio-browser-runner/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-browser-runner/node_modules/webdriverio -> ../../webdriverio
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-browser-runner/node_modules/devtools
wdio-browserstack-service
total 24K
drwxrwxr-x 4 4.0K Jan  4 20:13 cli
drwxrwxr-x 3 4.0K Jan  4 20:13 config
drwxrwxr-x 4 4.0K Jan  4 20:13 globals
lrwxrwxrwx 1   20 Jan  4 20:13 logger -> ../../../wdio-logger
drwxrwxr-x 3 4.0K Jan  4 20:13 protocols
drwxrwxr-x 3 4.0K Jan  4 20:13 repl
lrwxrwxrwx 1   22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1   19 Jan  4 20:13 types -> ../../../wdio-types
drwxrwxr-x 3 4.0K Jan  4 20:13 utils
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-browserstack-service/node_modules/webdriver
lrwxrwxrwx 1   17 Jan  4 20:13 wdio-browserstack-service/node_modules/webdriverio -> ../../webdriverio
drwxrwxr-x 4 4.0K Jan  4 20:13 wdio-browserstack-service/node_modules/devtools
wdio-cli
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1 21 Jan  4 20:13 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:13 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-cli/node_modules/webdriverio -> ../../webdriverio
wdio-concise-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-config
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
wdio-crossbrowsertesting-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-crossbrowsertesting-service/node_modules/webdriverio -> ../../webdriverio
wdio-cucumber-framework
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
wdio-devtools-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-devtools-service/node_modules/webdriverio -> ../../webdriverio
wdio-dot-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-firefox-profile-service
total 0
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-globals
total 28K
drwxrwxr-x 3 4.0K Jan  4 20:13 config
drwxrwxr-x 4 4.0K Jan  4 20:13 globals
drwxrwxr-x 4 4.0K Jan  4 20:13 logger
drwxrwxr-x 3 4.0K Jan  4 20:13 protocols
drwxrwxr-x 3 4.0K Jan  4 20:13 repl
drwxrwxr-x 3 4.0K Jan  4 20:13 types
drwxrwxr-x 3 4.0K Jan  4 20:13 utils
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-globals/node_modules/webdriver
lrwxrwxrwx 1   17 Jan  4 20:13 wdio-globals/node_modules/webdriverio -> ../../webdriverio
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-globals/node_modules/devtools
wdio-jasmine-framework
total 12K
drwxrwxr-x 4 4.0K Jan  4 20:13 config
lrwxrwxrwx 1   21 Jan  4 20:13 globals -> ../../../wdio-globals
lrwxrwxrwx 1   20 Jan  4 20:13 logger -> ../../../wdio-logger
drwxrwxr-x 3 4.0K Jan  4 20:13 protocols
drwxrwxr-x 3 4.0K Jan  4 20:13 repl
lrwxrwxrwx 1   19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1   19 Jan  4 20:13 utils -> ../../../wdio-utils
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-jasmine-framework/node_modules/webdriver
drwxrwxr-x 4 4.0K Jan  4 20:13 wdio-jasmine-framework/node_modules/webdriverio
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-jasmine-framework/node_modules/devtools
wdio-junit-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-local-runner
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 18 Jan  4 20:13 repl -> ../../../wdio-repl
lrwxrwxrwx 1 20 Jan  4 20:13 runner -> ../../../wdio-runner
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-logger
wdio-mocha-framework
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
wdio-protocols
wdio-repl
wdio-reporter
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-runner
total 8.0K
lrwxrwxrwx 1   20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1   21 Jan  4 20:13 globals -> ../../../wdio-globals
lrwxrwxrwx 1   20 Jan  4 20:13 logger -> ../../../wdio-logger
drwxrwxr-x 3 4.0K Jan  4 20:13 protocols
drwxrwxr-x 3 4.0K Jan  4 20:13 repl
lrwxrwxrwx 1   19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1   19 Jan  4 20:13 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:13 wdio-runner/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-runner/node_modules/webdriverio -> ../../webdriverio
drwxrwxr-x 3 4.0K Jan  4 20:13 wdio-runner/node_modules/devtools
wdio-sauce-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-sauce-service/node_modules/webdriverio -> ../../webdriverio
wdio-selenium-standalone-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-shared-store-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-shared-store-service/node_modules/webdriverio -> ../../webdriverio
wdio-smoke-test-cjs-service
wdio-smoke-test-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
wdio-smoke-test-service
wdio-spec-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-static-server-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-sumologic-reporter
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 22 Jan  4 20:13 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-testingbot-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-testingbot-service/node_modules/webdriverio -> ../../webdriverio
wdio-types
wdio-utils
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
wdio-webdriver-mock-service
total 0
lrwxrwxrwx 1 23 Jan  4 20:13 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:13 wdio-webdriver-mock-service/node_modules/webdriverio -> ../../webdriverio
webdriver
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:13 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
webdriverio
total 0
lrwxrwxrwx 1 20 Jan  4 20:13 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:13 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:13 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 18 Jan  4 20:13 repl -> ../../../wdio-repl
lrwxrwxrwx 1 19 Jan  4 20:13 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:13 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:13 webdriverio/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 14 Jan  4 20:13 webdriverio/node_modules/devtools -> ../../devtools
```

With these changes (all packages are symlinked):
```ls
devtools
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
eslint-plugin-wdio
node_modules
wdio-allure-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-appium-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-appium-service/node_modules/webdriverio -> ../../webdriverio
wdio-browser-runner
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 26 Jan  4 20:23 local-runner -> ../../../wdio-local-runner
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 29 Jan  4 20:23 mocha-framework -> ../../../wdio-mocha-framework
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 18 Jan  4 20:23 repl -> ../../../wdio-repl
lrwxrwxrwx 1 20 Jan  4 20:23 runner -> ../../../wdio-runner
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:23 wdio-browser-runner/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-browser-runner/node_modules/webdriverio -> ../../webdriverio
lrwxrwxrwx 1 14 Jan  4 20:23 wdio-browser-runner/node_modules/devtools -> ../../devtools
wdio-browserstack-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-browserstack-service/node_modules/webdriverio -> ../../webdriverio
wdio-cli
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-cli/node_modules/webdriverio -> ../../webdriverio
wdio-concise-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-config
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
wdio-crossbrowsertesting-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-crossbrowsertesting-service/node_modules/webdriverio -> ../../webdriverio
wdio-cucumber-framework
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
wdio-devtools-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-devtools-service/node_modules/webdriverio -> ../../webdriverio
wdio-dot-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-firefox-profile-service
total 0
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-globals
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1  5 Jan  4 20:23 globals -> ../..
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 18 Jan  4 20:23 repl -> ../../../wdio-repl
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:23 wdio-globals/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-globals/node_modules/webdriverio -> ../../webdriverio
lrwxrwxrwx 1 14 Jan  4 20:23 wdio-globals/node_modules/devtools -> ../../devtools
wdio-jasmine-framework
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 18 Jan  4 20:23 repl -> ../../../wdio-repl
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:23 wdio-jasmine-framework/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-jasmine-framework/node_modules/webdriverio -> ../../webdriverio
lrwxrwxrwx 1 14 Jan  4 20:23 wdio-jasmine-framework/node_modules/devtools -> ../../devtools
wdio-junit-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-local-runner
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 18 Jan  4 20:23 repl -> ../../../wdio-repl
lrwxrwxrwx 1 20 Jan  4 20:23 runner -> ../../../wdio-runner
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-logger
wdio-mocha-framework
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
wdio-protocols
wdio-repl
wdio-reporter
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-runner
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 18 Jan  4 20:23 repl -> ../../../wdio-repl
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:23 wdio-runner/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-runner/node_modules/webdriverio -> ../../webdriverio
lrwxrwxrwx 1 14 Jan  4 20:23 wdio-runner/node_modules/devtools -> ../../devtools
wdio-sauce-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-sauce-service/node_modules/webdriverio -> ../../webdriverio
wdio-selenium-standalone-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-shared-store-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-shared-store-service/node_modules/webdriverio -> ../../webdriverio
wdio-smoke-test-cjs-service
wdio-smoke-test-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
wdio-smoke-test-service
wdio-spec-reporter
total 0
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-static-server-service
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-sumologic-reporter
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 22 Jan  4 20:23 reporter -> ../../../wdio-reporter
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-testingbot-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-testingbot-service/node_modules/webdriverio -> ../../webdriverio
wdio-types
wdio-utils
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
wdio-webdriver-mock-service
total 0
lrwxrwxrwx 1 21 Jan  4 20:23 globals -> ../../../wdio-globals
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 17 Jan  4 20:23 wdio-webdriver-mock-service/node_modules/webdriverio -> ../../webdriverio
webdriver
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
webdriverio
total 0
lrwxrwxrwx 1 20 Jan  4 20:23 config -> ../../../wdio-config
lrwxrwxrwx 1 20 Jan  4 20:23 logger -> ../../../wdio-logger
lrwxrwxrwx 1 23 Jan  4 20:23 protocols -> ../../../wdio-protocols
lrwxrwxrwx 1 18 Jan  4 20:23 repl -> ../../../wdio-repl
lrwxrwxrwx 1 19 Jan  4 20:23 types -> ../../../wdio-types
lrwxrwxrwx 1 19 Jan  4 20:23 utils -> ../../../wdio-utils
lrwxrwxrwx 1 15 Jan  4 20:23 webdriverio/node_modules/webdriver -> ../../webdriver
lrwxrwxrwx 1 14 Jan  4 20:23 webdriverio/node_modules/devtools -> ../../devtools
```

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-browser-runner/package.json [hunk 1]
```diff
@@ -50,14 +50,14 @@
...
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
     "@types/ws": "^8.5.3",
-    "@wdio/runner": "8.1.0"
+    "@wdio/config": "8.1.0",
+    "@wdio/repl": "8.1.0",
+    "@wdio/runner": "8.1.0",
+    "devtools": "8.1.0"
```
Comment (christian-bromann) [inline packages/wdio-browser-runner/package.json:61]: Why is this needed when we have them in the `dependencies` section?

**Review by christian-bromann — CHANGES_REQUESTED** [review]
`scripts/depcheck.ts` and `scripts/generateSubPackage.js` seem to be empty files that were added

File: packages/wdio-browser-runner/package.json (hunk 1 above)
Comment (SCG82) [reply packages/wdio-browser-runner/package.json:61]: config, repl, runner, devtools aren't listed in dependencies for `wdio-browser-runner`. they are dependencies of:
```
    "@wdio/local-runner": "8.1.0",
    "@wdio/logger": "8.1.0",
    "@wdio/mocha-framework": "8.1.0",
    "@wdio/protocols": "8.1.0",
    "@wdio/types": "8.1.0",
    "@wdio/utils": "8.1.0",
    "webdriver": "8.1.0",
    "webdriverio": "8.1.0",
```
and apparently are installed at the top level of `wdio-browser-runner`'s node_modules (npm? lerna?), but not replaced with a symlink by lerna. they are also installed in the sub level `node_modules` (of each dependency) which is symlinked.

**Comment by SCG82** [comment]
> `scripts/depcheck.ts` and `scripts/generateSubPackage.js` seem to be empty files that were added

removed (I think they were added after this branch forked)

**Comment by SCG82** [comment]
I realized we can just delete those 😀

**Comment by SCG82** [comment]
ready for review. #9256 LGTM

**Review by christian-bromann — APPROVED** [review]
👍

**Comment by SCG82** [comment]
> 👍 (though I see the build broken)

Not sure why it broke when rebased onto main. Cool if I rebase it onto `cb-typescript-refactor`?

**Comment by SCG82** [comment]
Need to ensure that lerna bootstrap works correctly _every_ time.

**Comment by SCG82** [comment]
Better solution: https://github.com/webdriverio/webdriverio/pull/9623
